Lib/modulefinder.py

```diff
@@ -554,9 +554,9 @@ def replace_paths_in_code(self, co):
             self.processed_paths.append(original_filename)
 
         consts = list(co.co_consts)
```

Comment:
While we modify this function, I suggest to move type(co) out of the loop:

code_type = type(co)